File: _assets/output_gen/Entities/Question.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace OB.Data.Entities
{
    public class Question
    {
        //--------------------------------------------------------
        [Key]
        public int Id { get; set;}

        [Required, StringLength(2000)]
        public string Text { get; set;}

        public bool Yes { get; set;}

        public bool No { get; set;}

        //--------------------------------------------------------
    }
}

File: _assets/output_gen/jsModels/Question.js

//---------------------------------------------------------------------------

export default class Question {
  constructor(options) {

    this.Id = null;
    this.Text = null;
    this.Yes = null;
    this.No = null;

    if(options) {
      this.init(options);
    }
  }

  //---------------------------------------------------------------------------
  init(options) {

    this.Id = options.Id || this.Id;
    this.Text = options.Text || this.Text;
    this.Yes = options.Yes || this.Yes;
    this.No = options.No || this.No;

  }  
  //---------------------------------------------------------------------------  
}